web/shapeworks/src/views/Select.vue

```diff
@@ -165,7 +165,7 @@ export default defineComponent({
                     mdi-arrow-left
                 </v-icon>
             </div>
-            <v-card v-if="allProjectsForDataset.length === 0 && !loadingState" width="100%">
+            <v-card v-if="allProjectsForDataset === undefined && allProjectsForDataset.length === 0 && !loadingState" width="100%">
```

Comment:
I don't understand this change. Is there ever going to be a case where `allProjectsForDataset` is undefined AND its length is 0? Do you mean to use an OR?

Reply:
Seems like you're right. There won't ever be a case where those two conditions can be true simultaneously